dapp/src/components/Header/Navigation/Navigation.tsx

```diff
@@ -1,20 +1,29 @@
 import React from "react"
 import { Box, BoxProps, HStack, List } from "@chakra-ui/react"
-import { NavigationItemType } from "#/types/navigation"
-import NavigationItem from "./NavigationItem"
+import { EXTERNAL_HREF } from "#/constants"
+import { routerPath } from "#/router/path"
+import { isString } from "#/utils"
+import NavigationItem, { NavigationItemProps } from "./NavigationItem"
 
-type NavigationProps = BoxProps & {
-  items: NavigationItemType[]
-}
-
-function Navigation(props: NavigationProps) {
-  const { items, ...restProps } = props
+const NAVIGATION_ITEMS: NavigationItemProps[] = [
+  { children: "Home", to: EXTERNAL_HREF.WEBSITE, isExternal: true },
+  { children: "Stake", to: routerPath.home },
+  { children: "Docs", to: EXTERNAL_HREF.DOCS, isExternal: true },
+  { children: "FAQ", to: EXTERNAL_HREF.FAQ, isExternal: true },
+  { children: "Blog", to: EXTERNAL_HREF.BLOG, isExternal: true },
+  { children: "Discord", to: EXTERNAL_HREF.DISCORD, isExternal: true },
+  { children: "X", to: EXTERNAL_HREF.X, isExternal: true },
+]
 
+function Navigation(props: BoxProps) {
   return (
-    <Box as="nav" {...restProps}>
+    <Box as="nav" {...props}>
       <HStack as={List} spacing={5} ml={12}>
-        {items.map((item) => (
-          <NavigationItem key={item.href} {...item} />
+        {NAVIGATION_ITEMS.map((item) => (
+          <NavigationItem
+            key={isString(item.to) ? item.to : item.to.pathname}
```

Comment:
The type definition of `NAVIGATION_ITEMS` variable makes this line unnecessarily complicated.
The `to` type is always a `string` so the type check is redundant as you remove/simplify the type definition.

The given solution is much more readable in my opinion. The type definition is more specific and the need of programmatic type check disappears. The `label` is optional, you can keep `children` if you want but the `label` is more intuitive.

```tsx
type NavigationItemType = Omit<NavigationItemProps, 'children' | 'to'> {
  label: string
  to: string
}

const NAVIGATION_ITEMS: NavigationItemType[] = [
// ...
]

// ...

{NAVIGATION_ITEMS.map(({label, to, ...itemProps}) => (
  <NavigationItem  key={to} {...itemProps}>
    {label}
  </NavigationItem>
))}
```
 What do you think?

Reply:
> The type definition of NAVIGATION_ITEMS variable makes this line unnecessarily complicated.
The to type is always a string so the type check is redundant as you remove/simplify the type definition.

I would not limit the type for navigation items and let it be simply of type `NavigationItemProps`. This approach seems to me more intuitive and consistent with the rest instead of creating another slightly modified type that is used only in one place. 

I simplified the code a bit to make it more clear. - https://github.com/thesis/acre/pull/720/commits/a4c7ebeb8aea1bf96c7471dca9b8001c372f9087


> The given solution is much more readable in my opinion. The type definition is more specific and the need of programmatic type check disappears. The label is optional, you can keep children if you want but the label is more intuitive.

Declaring the `label` as a `string` limits later editing of navigation elements. If we want to add an icon to the navigation element, we will do it easily using `children`. Therefore, let's stay with the use of `children`.